figshare/Article.py

```diff
@@ -175,21 +184,19 @@ def private_article_for_data(self, private_url, article_id):
                                 version_metadata['errors'] = []
                                 version_metadata['errors'].append(error)
 
-                            self.total_all_articles_file_size += total_file_size
-
                             self.logs.write_log_in_file("info", f"{version_metadata} ")
 
                             error = f"{version_data['id']} - This item had a total embargo. The files are from version {version_data['version']}."
                             self.logs.write_log_in_file("info", f"{error}", True)
                             return version_data
                         else:
-                            error = f"{version_data['id']} - This item’s curation_status was not 'approved'"
+                            error = f"{version_data['id']} - This item’s curation_status was not 'approved'. It will be skipped during processing."
```

Comment:
The apostrophe in "item's" is not standard ASCII. It shows up garbled in certain text editors. Please fix here and other places where it may occur

Reply:
Updated in code